modules/data.land/R/put.veg.module.R

```diff
@@ -49,24 +35,29 @@
     logger.severe(paste(fcn, "does not exist."))
   }
   
+  # NOTE : match_pft is now called in veg2model
```

Comment:
It's been a while since we discussed this, but I thought we talked about a module that would both load data and match species before writing, and then a second module that would match PFTs and then call veg2model.